### ISSUE TITLE
light is not working

### ISSUE THREAD

**Issue by KShivendu** [haikarainen/light#89]
I am using i3 in Ubuntu and i tried to change sudo visudo to make light work without sudo after that it's not working at all. Please help

**Comment by haikarainen** [comment]
Im sorry to hear that you messed your system up. Try the udev rules method, its the recommended way for this project.

If you want further help from the community, Id recommend specifuing further details, such as the commands you have executed, and how you installed light.

**Comment by simon-n** [comment]
Despite the missing issue details, I just stumbled over the same issue when trying light on a Thinkpad P1. 
Solution was to install the udev rules file into /etc/udev/rules.d in order to grant write permissions to "video"-group members.

**Comment by haikarainen** [comment]
@simon-n as per the Readme then. What distro are you on and how did you install light?

**Comment by helloWorld670** [comment]
Today morning light started working when i reinstalled it from .deb file through ubuntu instead of i3.

Thank you very much for your active response sir.

**Comment by simon-n** [comment]
@haikarainen I'm on Ubuntu 18.04 and installed light_1.2_amd64.deb. Did I miss a Ubuntu-specific build on the releases page?

**Comment by haikarainen** [comment]
@simon-n not sure if we even have a ubuntu maintained pavkage. @troglobit ?

**Comment by troglobit** [comment]
There is a .deb file I uploaded to the releases page along with the tarball.  It was built on Ubuntu 18.10 x86_64 and should work as-is on 18.10 and later Ubuntu releases. It may also work on newer Debian systems, and other Debian derived distros, as long as they have the same or newer GLIBC version.

There is actually an official Debian package of light coming, which will also make its way into Ubuntu.  The packaging we did was used as an example of how to package for Debian and uploaded for the next Debian stable cycle (this current cycle has just closed unfortunately).

So, @simon-n the package you've downloded may work also on 18.04.
